src/components/tables/StorageMappings/index.tsx

```diff
@@ -43,6 +43,7 @@ function StorageMappingsTable({ header }: Props) {
     return (
         <Box>
             <TableHydrator
+                disableQueryParamHack
```

Comment:
This table does open with navigation but still got messed up with the hack so just switching off.